### PR TITLE
Fix jitsi widget sometimes being permanently stuck in the bottom-right corner

### PR DESCRIPTION
**Pull request by Palid**
Fixes https://github.com/vector-im/element-web/issues/17226

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix jitsi widget sometimes being permanently stuck in the bottom-right corner ([\#6632](https://github.com/matrix-org/matrix-react-sdk/pull/6632)). Fixes vector-im/element-web#17226 and vector-im/element-web#17226. Contributed by [Palid](https://github.com/Palid).

Preview: https://611de22077a7388d9d1902da--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/elements/AppTile.js [hunk 1]
```diff
@@ -443,25 +443,25 @@ export default class AppTile extends React.Component {
...
+                            /> }
+                            <ContextMenuButton
+                                className="mx_AppTileMenuBar_iconButton mx_AppTileMenuBar_iconButton_menu"
+                                label={_t("Options")}
+                                isExpanded={this.state.menuDisplayed}
+                                inputRef={this._contextMenuButton}
+                                onClick={this._onContextMenuClick}
+                            />
+                        </span>
+                    </div> }
```
Comment (dbkr) [inline src/components/views/elements/AppTile.js:464]: Best to separate the indenting changes out into a different commit

File: src/components/views/elements/AppTile.js [hunk 2]
```diff
@@ -307,7 +308,6 @@ export default class AppTile extends React.Component {
                 if (this.iframe) {
                     // Reload iframe
                     this.iframe.src = this._sgWidget.embedUrl;
-                    this.setState({});
```
Comment (dbkr) [inline src/components/views/elements/AppTile.js:310]: Was this change related or fixing a different bug?

**Review by dbkr — COMMENTED** [review]
Also don't forget your PR title will be what goes in the changelog, so it should user-facing rather than dev-facing, or if you want a dev-facing PR title, use `notes:`. And this definitely counts as a defect as this was a bug lots of people noticed.

File: src/components/views/elements/AppTile.js (hunk 2 above)
Comment (Palid) [reply src/components/views/elements/AppTile.js:310]: This didn't do anything due to shallow merge, so no idea why it was even here. To force rerender (which didn't work btw)?

File: src/components/views/elements/AppTile.js (hunk 1 above)
Comment (Palid) [reply src/components/views/elements/AppTile.js:464]: @dbkr split&force pushed, try now.

File: src/components/views/elements/AppTile.js (hunk 2 above)
Comment (Palid) [reply src/components/views/elements/AppTile.js:310]: Split it into a separate commit just in case.

**Review by dbkr — APPROVED** [review]
